test/relayserver/RelayServer.test.ts

```diff
@@ -484,58 +481,6 @@ describe('RelayServer', function () {
       });
     });
 
-    describe('maxPossibleGasWithViewCall', function () {
```

Comment:
Why did we remove this test?

Reply:
this function does not longer exists in the server, I can move it to the server, since I dont think that removing it is the best choice.

Reply:
updated, I return the function to avoid breaking changes.